torchtext/csrc/regex.cpp

```diff
@@ -1,26 +1,15 @@
-#include <re2/re2.h>
-#include <string>
-#include <torch/script.h>
+#include <regex.h>
```

Comment:
What are those two files `regex.cpp` and `regex.h` for? Where are we use them?

Reply:
Basically I was thinking of removing the `Regex` class  now since we aren't using it anymore. But @cpuhrsch suggested doing this in a followup PR once we've had more time to evaluate whether we want a light wrapper around the `RE2` library for future use cases. Does that sound reasonable?

Reply:
Sounds good. Let's keep this as the plan.